apps/emqx/src/emqx_schema.erl

```diff
@@ -1374,6 +1383,16 @@ to_comma_separated_list(Str) ->
 to_comma_separated_atoms(Str) ->
     {ok, lists:map(fun to_atom/1, string:tokens(Str, ", "))}.
 
+rfc3339_to_system_time(DateTime) ->
+    try
+        {ok, calendar:rfc3339_to_system_time(DateTime, [{unit, second}])}
+    catch error: {badmatch, _} ->
+        {error, ["The rfc3339 specification not satisfied"]}
```

Comment:
why return a list of strings here ?
```suggestion
        {error, ["bad_rfc3339_timestamp"]}
```

Reply:
or maybe `{error, bad_rfc3339_timestamp}`

Reply:
good catch! `{error, bad_rfc3339_timestamp}` is better.